src/components/FilterDropdown.tsx

```diff
@@ -1,4 +1,10 @@
 import { Dispatch, SetStateAction, useMemo, useState } from "react";
+import { library } from "@fortawesome/fontawesome-svg-core";
+import { fas } from "@fortawesome/free-solid-svg-icons";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+
+library.add(fas);
```

Comment:
nit: This is definitely one way to approach using `FontAwesomeIcon`. In terms of code grouping, I don't think it makes sense to add `fas` to a list of icons in `FilterDropDown`. Instead of

`<FontAwesomeIcon icon="caret-down" className=" text-dark-teal" />`

I would opt-in for

`<FontAwesomeIcon icon={faCaretDown} className=" text-dark-teal" />